html/static/script.js

```diff
@@ -73,3 +73,25 @@ $(document).ready(function() {
     $('body').addClass('dev-ribbon');
   }
 });
+
+function addSample(element) {
+    console.log($(element).parent().parent().parent());
+    let datasetname = $(element).parent().parent().parent().find('#datasetnamerun3').get(0).value;
+    console.log(datasetname);
+    let numberevents = $(element).parent().parent().parent().find('#numberofevents').get(0).value;
+    console.log(numberevents);
+    let interested_pwgs = $(element).parent().parent().parent().find('#pwginterested').get(0).value;
+    console.log(interested_pwgs);
+    let data = {"datasetname": datasetname, "numberofevents": numberevents, "pwginterested": interested_pwgs};
+    $.ajax({
+        type: "POST",
+        contentType: "application/json",
+        url: "/add_run3",
+        data: JSON.stringify(data),
+    }).done(function (data) {
+        alert('Text saved in the database');
+    }).fail(function(data) {
+        alert('Error in saving');
```

Comment:
Add `data` to the error message. If server returns an error message, show it to the user.